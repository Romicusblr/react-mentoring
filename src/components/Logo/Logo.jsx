import React from 'react';
import { Link } from 'react-router-dom';
import classNames from 'classnames';
import PropTypes from 'prop-types';
import style from './Logo.module.css';

function Logo({ className, ...other }) {
  return (
    <Link
      to="/"
      className={classNames(style.logo, className)}
      {...other}
    >
      <h3>
        <strong>netflix</strong>
      roulette
      </h3>
    </Link>
  );
}

Logo.propTypes = {
  className: PropTypes.string,
};

Logo.defaultProps = {
  className: '',
};

export default Logo;
