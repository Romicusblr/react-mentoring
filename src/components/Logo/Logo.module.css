.logo {
  outline: 0;
  text-decoration:none;
  &>h3 {
    color: var(--color-raised);
    font-weight: 100;
    &>strong {
      font-weight: 900;
    }
  }
}
