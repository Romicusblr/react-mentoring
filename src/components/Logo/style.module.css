.logo {
  color: var(--color-pink);
  font-weight: 100;
  &>strong {
    font-weight: 900;
  }
}
