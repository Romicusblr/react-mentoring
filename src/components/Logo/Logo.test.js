import Logo from '.';

describe('Logo', () => {
  it('should match to snapshot', () => {
    const snap = shallow(<Logo />);
    expect(snap).toMatchSnapshot();
  });
});
