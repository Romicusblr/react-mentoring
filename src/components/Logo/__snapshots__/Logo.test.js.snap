// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match to snapshot 1`] = `
<h3
  className="logo"
>
  <strong>
    netflix
  </strong>
  roulette
</h3>
`;
